src/modules/shopPage/components/FilterMenu/FilterFullMenu/FilterFullMenuContainer.tsx

```diff
@@ -0,0 +1,285 @@
+import * as React from "react";
+import { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { selectActiveTabIndex } from "../../../selectors";
+import { actions } from "../../../reducer";
+
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import { IconButton, styled } from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+
+import FilterFullMenuInput from "./FilterFullMenuInput";
+import FilterFullMenuWidthData from "./FilterFullMenuWidthData";
+import FilterFullMenuProfileData from "./FilterFullMenuProfileData";
+import FilterFullMenuDiametrData from "./FilterFullMenuDiametrData";
+import FilterFullMenuPriceData from "./FilterFullMenuPriceData";
+import FilterFullMenuSeasonData from "./FilterFullMenuSeasonData";
+import FilterFullMenuBrandData from "./FilterFullMenuBrandData";
+
+import WidthIcon from "../../../../../shared/components/Icons/WidthIcon";
+import ProfileIcon from "../../../../../shared/components/Icons/ProfileIcon";
+import SeasonIcon from "../../../../../shared/components/Icons/SeasonIcon";
+import BrandIcon from "../../../../../shared/components/Icons/BrandIcon";
+import PriceIcon from "../../../../../shared/components/Icons/PriceIcon";
+import DiametrIcon from "../../../../../shared/components/Icons/DiametrIcon";
+
+import FilterFullMenuData from "../../../../../config/FilterFullMenuData.json";
+import { FONTS, BASE_COLORS } from "../../../../../shared/constants";
+
+type TabPanelProps = {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+};
+
+type FilterData = {
+  headerTitle?: string;
+  asideHeader?: string;
+  text1?: string;
+  text2?: string;
+  text3?: string;
+  parametr?: string;
+  textForParametr?: string;
+};
+
+type FilterFullMenuDataType = {
+  Width?: FilterData;
+  Profile?: FilterData;
+  Diametr?: FilterData;
+  Price?: FilterData;
+  Season?: FilterData;
+  Brand?: FilterData;
+};
+
+type MenuData = {
+  inputComponent?: React.ElementType;
+  dataComponent: React.ElementType;
+} & FilterData;
+
+const StyledText = styled(Typography)({
+  fontSize: "14px",
+  marginTop: "18px",
+  fontFamily: FONTS.MAIN_TEXT_FAMILY,
+});
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
```

Comment:
use MUI components instead of default HTML components

Reply:
fixed it in all components and deleted extra lines